index.js

```diff
@@ -72,3 +132,11 @@ function isAllowPublic (input) {
 
   return true
 }
+
+function isAllowPublic2 (input) {
+  if (typeof input.content !== 'object') return false
+  if (typeof get(input, ['content', 'type']) !== 'string') return false
+  if (get(input, ['options', 'allowPublic']) !== true) return false
```

Comment:
Needs the README updating to show how to use it with db2. this signature is different?
Also is `options` used by anything else ? https://www.npmjs.com/package/ssb-db2#createopts-cb

Doesn't seem like it. In that case make it `input.allowPublic` I think.

Reply:
updated readme